server/cmd/command.go

```diff
@@ -345,3 +335,41 @@ func exportedFields(command reflect.Value) []reflect.StructField {
 	}
 	return fields
 }
+
+// SplitCommandArgs splits str to args for command execution.
+func SplitCommandArgs(str string) []string {
```

Comment:
Function should not be exported.

Reply:
I realise that this is used in player.go, but we should probably adapt that in one way or another so that this does not require being exported.